web/src/components/commits_timeline.js

```diff
@@ -0,0 +1,50 @@
+// Monocle.
+// Copyright (C) 2020 Monocle authors
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Timeline, TimelineEvent } from 'react-event-timeline'
+import moment from 'moment'
+
+class CommitsTimelineGraph extends React.Component {
+  getTitle (commit) {
+    return commit.title
+  }
+
+  getDate (commit) {
+    return moment(commit.authored_at).fromNow() + ' by ' + commit.author
+  }
+
+  render () {
+    console.log(this.props.data)
```

Comment:
Maybe just remove this console.log. I approve so let's merge after that removal.